mods/GooseManager.md

```diff
@@ -0,0 +1,47 @@
+# 🙂 GooseManager
+
+## Features
+
+Adds a file where you can disable goose actions
+
+Adds a file where you can disable goose speed tiers
+
+## How To Use
+
+**disabling goose actions:**
+
+After first time of starting goose with this mod, "GooseTasks.txt" file will be added to folder, there you can disable action by changing True to False on the right side of action name!
+
+If all actions are disabled, all actions will be enabled
+
+**disabling goose speed tiers:**
+
+After first time of starting goose with this mod, "GooseSpeedTiers.txt" file will be added to folder, there you can disable speed tier by changing True to False on the right side of speed tier!
+
+If all speed tiers are disabled, all speed tiers will be enabled
+
+**for both:**
+
+To apply changes you need to restart goose
+
+## Attention
+
+Having only charge active while having active meme and note actions will make goose dancing with window while holded window is on
+
+Disabling charge will not change goose speed in Nabmouse action (when goose catches mouse speed will change)
+
+## Info
+
+Get it: [GooseManager.dll](https://github.com/1JQ/GooseManager/blob/master/For%20v0.3/GooseManager.dll?raw=true )
+
+Goose Version: **v0.3**
+
+Mod Version: **v1.0**
+
+Author: **JQ#2077**
+
+Credit For Name: **Moracan#3634**
+
+Source Code: [SourceCode.cs](https://github.com/1JQ/GooseMenager/blob/master/For%20v0.3/SourceCode.cs )
+
+{% include install_guide.md modname="GooseMenager" iszip=false %}
```

Comment:
Also, this uses the wrong DLL file name:
```suggestion
{% include install_guide.md modname="GooseManager" iszip=false %}
```

Reply:
Whoops I forgot about that